ui/pages/notifications/notification-components/feature-announcement/feature-announcement.tsx

```diff
@@ -95,8 +96,9 @@ export const components: NotificationComponent<FeatureAnnouncementNotification>
             <Text
               variant={TextVariant.bodyMd}
               as="div"
+              // TODO - we can replace the raw HTML string injection with react components
```

Comment:
I think we can avoid string injection by using the contentful react renderer:
https://github.com/contentful/rich-text/tree/master/packages/rich-text-react-renderer

This package is pretty small, so I think we can copy this in CORE to be shareable with React/React-Native.

Reply:
Hmm I don't know if we can use `React` as a dev dependency in CORE.
We can tweak the interface to be JSX agnostic (that way we can inject react components or react-native components)